extensions/Squidex.Extensions/Actions/Email/EmailActionHandler.cs

```diff
@@ -0,0 +1,74 @@
+﻿// ==========================================================================
+//  Squidex Headless CMS
+// ==========================================================================
+//  Copyright (c) Squidex UG (haftungsbeschraenkt)
+//  All rights reserved. Licensed under the MIT license.
+// ==========================================================================
+
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Mail;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Options;
+using Squidex.Domain.Apps.Core.HandleRules;
+using Squidex.Domain.Apps.Core.HandleRules.EnrichedEvents;
+using Squidex.Infrastructure;
+
+namespace Squidex.Extensions.Actions.Email
+{
+    public sealed class EmailActionHandler : RuleActionHandler<EmailAction, EmailJob>
+    {
+        private const string Description = "Send an Email";
+
+        private readonly EmailOptions emailOptions;
+
+        public EmailActionHandler(RuleEventFormatter formatter, IOptions<EmailOptions> emailOptions)
+            : base(formatter)
+        {
+            Guard.NotNull(emailOptions, nameof(emailOptions));
+            this.emailOptions = emailOptions.Value;
+        }
+
+        protected override (string Description, EmailJob Data) CreateJob(EnrichedEvent @event, EmailAction action)
+        {
+            var ruleJob = new EmailJob
+            {
+                From = action.From,
```

Comment:
I think from and to should be also formattable, e.g. you could create the email to the user who created a content.